### ISSUE TITLE
Areas of the features are not exported to the GNPS_quan file

### ISSUE THREAD

**Issue by silvinaperin** [mzmine/mzmine3#1802]
Hello,

I have the following issue:

From the visualization of the alignment in the MZmine I can see that one specific feature (765; I am particularly interested in it) was integrated and has an area for the 4 samples I have processed

![Capture](https://github.com/mzmine/mzmine3/assets/163825843/e9860667-8315-4722-9ace-042c10470c8e)

However in the quan.csv file, if I look for these areas, the cells are empty. I am wondering if the exporting process is performing correctly. Please find attached the export file of the csv
(It is not that there are not areas at all. You can see areas of others features in the file.)

[MZmine_results_iimn_gnps_quant.csv](https://github.com/mzmine/mzmine3/files/15022311/MZmine_results_iimn_gnps_quant.csv)

Thanks!!

**Comment by SteffenHeu** [comment]
![grafik](https://github.com/mzmine/mzmine3/assets/37407705/f98c919d-2708-4333-9531-0c8225fe5862)

looks ok to me when searching for ID 765 in the editor

Is excel maybe doing some weird stuff when opening the file?

**Comment by silvinaperin** [comment]
Hi Steffen, 

I have checked again without formatting the file into columns and I realized that when I open the file as it is, there are more than one column: 



so, in this case I do not know how to transform this into a table with columns. First column contains the info and the rest are empty in the header. 


is there any setting in the exporting that maybe I am not doing correctly and therefore I am getting more than one column? 

Thanks 

Silvina
